lib/proto/xudrpc.swagger.json

```diff
@@ -21,7 +21,7 @@
         "operationId": "AddCurrency",
         "responses": {
           "200": {
-            "description": "",
+            "description": "A successful response.",
```

Comment:
Looks like you have a different `protoc-gen-swagger` from what was used last, I need to update our instructions to use a specific version but for now can you try this and then regenerating protos?

```
git clone https://github.com/grpc-ecosystem/grpc-gateway $GOPATH/src/github.com/grpc-ecosystem/grpc-gateway
cd $GOPATH/src/github.com/grpc-ecosystem/grpc-gateway
git reset --hard f2862b476edcef83412c7af8687c9cd8e4097c0f
go install ./protoc-gen-swagger
```